### PR TITLE
Core page wording and flow

### PR DESCRIPTION
**Pull request by hughns**
Remove redundant buttons/links
Make wording more directive
Describe scope of access

### CODE REVIEW

File: crates/templates/src/res/account-base.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+#}
+
+{% extends "base.html" %}
+
+{% block content %}
+<nav class="container mx-auto py-2 flex-initial flex items-center px-2" role="navigation" aria-label="main navigation">
```
Comment (sandhose) [inline crates/templates/src/res/account-base.html:20]: I think it makes more sense to have a partial/component and `include` it

File: crates/templates/src/res/pages/login.html [hunk 2]
```diff
@@ -33,31 +33,33 @@
...
       {# TODO: errors #}
       {{ field::input(label="Username", name="username") }}
       {{ field::input(label="Password", name="password", type="password") }}
-      {{ button::button(text="Next") }}
-      {{ button::link_text(text="Create account", href=register_link) }}
+      {% if next and next.kind == "continue_authorization_grant" %}
+        <div class="grid grid-cols-2 gap-4">
+          {{ back_to_client::link(
+            text="Cancel",
+            class=button::text_class(),
```
Comment (sandhose) [inline crates/templates/src/res/pages/login.html:47]: `class=button::plain_error_class()` would probably make it prettier

Comment (sandhose) [inline crates/templates/src/res/pages/login.html:47]: `class=button::plain_error_class()` would probably make it prettier

**Review by sandhose — CHANGES_REQUESTED** [review]

File: crates/templates/src/res/pages/login.html (hunk 2 above)
Comment (hughns) [reply crates/templates/src/res/pages/login.html:47]: I think I changed it from the solid red/alert block to the text as I felt it was distracting from the Next action.

How about this instead?

image

**Review by sandhose — APPROVED** [review]

**Pull request merged**